### DISCUSSION TITLE
To make two cabinets with laravel backpack 6 and another db tables

### DISCUSSION THREAD

**Discussion by k1g0r** [Laravel-Backpack/community-forum#681] [Q&A (Help)] [answered]
In my project I need to create an admin panel and a client panel. For security I want to store admins and clients in different database tables (admins | clients) For these purposes I am going to use laravel backpack, but the question is how to configure backpack so that I have two separate authorization links /admin and /client with design and full support for backpack

I tried to solve the problem using modularity nWidart/laravel-modules but it didn’t work, the routes from the backpack were not connected

**Comment by karandatwani92** [comment, marked as answer]
For the admin panel(BACKPACK), you can set different model in `config/backpack/base.php`
```php
'user_model_fqn' => App\Models\Admin::class,
```

For the client panel(NOT BACKPACK), you can keep using the user model which is set in `config/auth.php`
```php
'users' => [
            'driver' => 'eloquent',
            'model' => App\Models\User::class,
        ],
```

But you can't use different model/table for each role of the admin panel.

More Info: https://backpackforlaravel.com/docs/6.x/base-about#using-a-different-user-model-1

**Reply by k1g0r** [reply]
it's ok. But i want to use backpack for admin panel and client panel.
